test/e2e/e2e.sh

```diff
@@ -470,8 +473,37 @@ then
   exit 1
 fi
 
+# Webhook should reject invalid resources (v1beta1 version)
 if ! kubectl create --dry-run=server -f- <<'EOF' 2>&1 | grep -qF 'unknown time-slice interval'
 apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaim
+metadata:
+  name: webhook-test-v1beta1
+spec:
+  devices:
+    requests:
+    - name: ts-gpu
+      exactly:
+        deviceClassName: gpu.example.com
```

Comment:
I think e2e is failing because this isn't an otherwise valid v1beta1 ResourceClaim.

```suggestion
      deviceClassName: gpu.example.com
```

Reply:
Yes just figured that out locally fixing now thank you